Library/Homebrew/brew.sh

```diff
@@ -359,6 +359,18 @@ fi
 ##### Now, do everything else (that may be a bit slower).
 #####
 
+# Docker image deprecation
+if [[ -f "${HOMEBREW_REPOSITORY}/.docker-deprecate" ]]
```

Comment:
How about putting this in the docker root rather than repository root?

Reply:
Will require modifying the Dockerfile, I think. Could make it an arg that gets passed that, if set, does this.